Lib/collections/__init__.py

```diff
@@ -636,7 +642,7 @@ def update(*args, **kwds):
             raise TypeError('expected at most 1 arguments, got %d' % len(args))
         iterable = args[0] if args else None
         if iterable is not None:
-            if isinstance(iterable, Mapping):
+            if isinstance(iterable, _collections_abc.Mapping):
```

Comment:
Does this change have an impact on performance? If yes, maybe we should keep a _Mapping global?

Reply:
I don't know. In any case the isinstance check is not cheap for ABCs. I suppose that one attribute resolution has insignificant effect on performance.